package.json

```diff
@@ -91,72 +91,41 @@
   "devDependencies": {
     "@babel/compat-data": "^7.9.0",
     "@mongodb-js/sbom-tools": "^0.5.2",
-    "@octokit/rest": "^17.9.0",
-    "@types/bl": "^2.1.0",
     "@types/chai": "^4.2.5",
-    "@types/download": "^8.0.1",
     "@types/mocha": "^5.2.7",
     "@types/node": "^14.14.6",
-    "@types/node-fetch": "^2.5.7",
     "@types/rimraf": "^3.0.0",
     "@types/semver": "^7.3.4",
     "@types/sinon-chai": "^3.2.4",
-    "@types/tar": "^4.0.3",
-    "@types/tar-fs": "^2.0.0",
-    "@types/tmp": "^0.2.0",
     "@types/which": "^1.3.2",
     "@typescript-eslint/eslint-plugin": "^4.28.4",
     "@typescript-eslint/parser": "^4.28.4",
-    "aws-sdk": "^2.674.0",
-    "axios": "^0.21.1",
-    "boxednode": "^2.0.1",
-    "browserify": "^16.5.0",
     "chai": "^4.2.0",
-    "command-exists": "^1.2.9",
     "cross-env": "^6.0.3",
-    "depcheck": "^1.4.1",
-    "download": "^8.0.0",
     "duplexpair": "^1.0.1",
     "eslint": "^7.31.0",
     "eslint-config-mongodb-js": "^5.0.3",
     "eslint-plugin-mocha": "^9.0.0",
     "find-up": "^5.0.0",
-    "handlebars": "^4.7.7",
-    "js-yaml": "^3.13.1",
-    "karma": "^6.3.4",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-cli": "^2.0.0",
-    "karma-mocha": "^2.0.1",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-typescript": "^5.5.1",
     "lerna": "^4.0.0",
     "mocha": "^10.2.0",
     "mongodb": "^5.6.0",
```

Comment:
This is used by the scripts in `testing/`, we may want to turn this into a proper package (but preferably wait with that until the mongodb-runner work is done?)